projects/wave-core-new/src/lib/colors/color-attribute-input/color-attribute-input.component.scss

```diff
@@ -2,7 +2,7 @@
     font-size: 0.8rem;
 }
 
-mat-form-field >>> .color_cell {
+mat-form-field ::ng-deep .color_cell {
```

Comment:
You missed another one:

```
⠼ Compiling with Angular sources in Ivy full compilation mode.WARNING: ▲ [WARNING] Unexpected ">"

    /home/beilschmidt/git/geoengine-ui/projects/wave-core-new/src/lib/colors/color-breakpoint-input/color-breakpoint-input.component.scss:5:17:
      5 │ mat-form-field > > > .color_cell {
        ╵                  ^

```

Reply:
yes i do not know how to solve this. If you remove the ::ng-deep the color input will  miss the white border on text. But there is no suggestion what would fix this. The entire /deep/ >>> ::ng-deep api  is deprecated anyway and will be removed in the future.

Reply:
fbd735e